animal-sniffer-signature/build.gradle.kts

```diff
@@ -24,14 +24,11 @@ val signatureJarClasspath =
         isCanBeResolved = true
         extendsFrom(signatureJar)
     }
-val generatedSignature =
-    configurations.create("generatedSignature") {
-        isCanBeConsumed = true
-        isCanBeResolved = false
-    }
-configurations.add(signatureJar)
-configurations.add(signatureJarClasspath)
-configurations.add(generatedSignature)
```

Comment:
The javadocs for `create()` say that it also adds it to the container, so it seems like these were redundant/unnecessary. At least in gradle 9 it's a build failure.

Reply:
Thanks for taking a look!